code/controllers/subsystem/traumas.dm

```diff
@@ -45,6 +46,7 @@ SUBSYSTEM_DEF(traumas)
 					   /mob/living/simple_animal/revenant, /mob/living/simple_animal/shade)),
 					   "aliens"   = typecacheof(list(/mob/living/carbon/alien, /mob/living/simple_animal/slime)),
 					   "conspiracies" = typecacheof(list(/mob/living/simple_animal/bot/secbot, /mob/living/simple_animal/bot/ed209, /mob/living/simple_animal/drone))
+					   "birds" = typecacheof(list(/mob/living/simple_animal/parrot))
```

Comment:
Chickens, ducks, penguins

Reply:
oh wait ducks don't exist
Also the chicken suit and optionally chicken nuggets.